androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/ControlsDecoration.kt

```diff
@@ -0,0 +1,15 @@
+package com.profusion.androidenhancedvideoplayer.components
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+data class ControlsDecoration(
```

Comment:
Is Decoration here about the decorator pattern? From the content, it is not. I'd choose another naming. This seems to be more of a style modifier, what do you think about `PlayerControlsStyle`?

Reply:
Is there a specific reason for it to be a data class? Are we planning on using any of the methods that come with it? (`equals()`, `toString()`, `copy()`, etc.)

Reply:
Also, I'd not create another file under `components/`. My idea is to keep a 1:1 map between a file in the `components/` folder and an actual composable that can be embedded into screens. Instead, this seems that can be defined inside `PlayerControls.kt` as it is specific to that component.